src/services/index.ts

```diff
@@ -438,12 +438,12 @@ export default class Provider extends CloudGraph.Client {
    * getSchema is used to get the schema for provider
    * @returns A string of graphql sub schemas
    */
-  getSchema(): string {
+  getSchema(): any {
```

Comment:
are we able to type this?

Reply:
The type is DocumentNode. The problem here is that the aws provider uses an older version of the graphql package. And that type is not compatible with the newer versions of graphql used by the other providers and the sdk. I tried to update the package in the aws provider but it brings another dependency errors. That's why I chose to set the type any.

Reply:
hmm what are the other dep errors? we should try to update the version of the package if possible

Reply:
can we update the version of the graphql code gen? He says the latest release for that library (which I assume is a dep of graphql code gen typescript lib) might fix the issue.

Reply:
done. it's already fixed!